_posts/fr/2022-03-22-enquete-sur-le-crypto-gaming.md

```diff
@@ -87,25 +87,19 @@ La réponse ne me plaît pas : un jeton est créé à partir de rien, il n'a de
 
 Comme pour la crypto-monnaie classique, les tokens possèdent leur propre cours boursier.
 
-<div style="text-align: center;">
-    <img src="{{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/axie-slp.png" width="700px" alt="Cours de l'objet SLP du jeu Axie Infinity" style="display: block; margin: auto;"/>
-</div>
+![Cours de l'objet SLP du jeu Axie Infinity]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/axie-slp.png)
```

Comment:
width missing

Reply:
I did not put it because the difference is hardly visible visually